setup.cfg

```diff
@@ -32,10 +32,6 @@ install_requires =
     python-dateutil >= 2.7.0
     types-python-dateutil >= 2.7.0
     netCDF4 >= 1.6.0
```

Comment:
IMO `pystac`, at least, should be an explicit dependency, because we use it directly in code.

Reply:
done.